tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -289,13 +289,14 @@ uint32_t TypeTranslator::getGlPerVertexStruct(uint32_t clipArraySize,
   auto &ctx = *theBuilder.getSPIRVContext();
   llvm::SmallVector<const Decoration *, 1> decorations;
 
-  decorations.push_back(Decoration::getBuiltIn(ctx, spv::BuiltIn::Position, 0));
-  decorations.push_back(
-      Decoration::getBuiltIn(ctx, spv::BuiltIn::PointSize, 1));
-  decorations.push_back(
-      Decoration::getBuiltIn(ctx, spv::BuiltIn::ClipDistance, 2));
-  decorations.push_back(
-      Decoration::getBuiltIn(ctx, spv::BuiltIn::CullDistance, 3));
+  decorations.push_back(Decoration::getBuiltIn(ctx, spv::BuiltIn::Position,
+                                               llvm::Optional<uint32_t>(0)));
```

Comment:
Bad rebasing? `llvm::Optional<..>()` should already be removed.

Reply:
This is odd! fixed.